systemvm/debian/opt/cloud/bin/setup/secstorage.sh

```diff
@@ -80,10 +80,22 @@ CORS
   rm -f /etc/logrotate.d/cloud
 }
 
+import_jvm_cacerts(){
+ log_it "importing jvm keystore to realhostip keystore"
+ keyStore=/usr/local/cloud/systemvm/certs/realhostip.keystore
+ storepass="vmops.com"
+ java_home=$(readlink -f $(which java) | sed "s:/bin/java::")
+ java_keystore=$java_home/jre/lib/security/cacerts
```

Comment:
Tested manually, the keystore path should be `$java_home/lib/security/cacerts` for 4.11/debian9 based systemvmtemplate. Also, consider using upper case variables such as `JAVA_HOME`, `JAVA_KEYSTORE` etc.